### PR TITLE
Increment version to 1.1.7

### PR DESCRIPTION
**Pull request by michaelpnelson**

### CODE REVIEW

**Review by rustyjux — APPROVED** [review]
Not much to reviewing version increments!

**Comment by michaelpnelson** [comment]
FYI I won't merge this right away. I might be able to merge one or two other features before we merge this and do a release. But it's great to have this approved now so that I can release when I need to.

**Comment by rustyjux** [comment]
Now that seems like a sneaky way to slip in features without my careful review!

**Pull request merged**
